tasks/config.yml

```diff
@@ -59,6 +59,25 @@
     force: true
   notify: restart SolrCloud
 
+- name: SolrCloud | Install download war from maven require package
```

Comment:
If I understood this well, I think this task name would be a little more accurate: "`Install maven_artifact` module required packages"

Reply:
done